### ISSUE TITLE
wrong unformatted paste behavior

### ISSUE THREAD

**Issue by FranckFreiburger** [quilljs/quill#2652]
**Steps for Reproduction**

1. Visit quilljs.com
2. Copy some text (Ctrl-C)
3. Place the caret in a link text
4. Press: Ctrl-Shift-V (Option+Shift+Command+V on MacOS)

**Expected behavior**:

The pasted text to be inserted without breaking the format at caret position

**Actual behavior**:

The pasted text is properly inserted at the caret position but breaks the format

**Platforms**:
Win7, chrome 75

**Version**:
Quill.version: 1.3.6

**Comment by FranckFreiburger** [comment]
Possible workaround:

```
quill.keyboard.addBinding({
  key: 'V',
  ctrlKey: true,
  shiftKey: true,
}, function(range, context) {

	if ( !navigator.clipboard )
		return;
	navigator.clipboard.readText().then(clipText => this.quill.insertText(range.index, clipText));
});

```

**Comment by volser** [comment]
You can implement your own clipboard watcher
https://quilljs.com/docs/modules/clipboard/
and return Delta without any format

**Comment by FranckFreiburger** [comment]
Thanks @volser.
Do you have an idea of how to detect that the clipboard operation come from an "unformatted paste" instead of a "normal paste",
Watcher callback only provides node and delta arguments, nothing about the source of the event.

**Comment by FranckFreiburger** [comment]
In fact, unformatted paste is supported but the behavior is not the expected one.

https://github.com/quilljs/quill/blob/2b2fb56c526a4d07a11180b6d518ee41c6e60b7c/modules/clipboard.js#L85-L87

If the html version is not present (unformatted text) the text version is inserted, but without format.
The behavior become correct when adding `formats` (just defined above)

```js
return new Delta().insert(text || '', formats);
```

**Comment by luin** [comment]
Will be fixed in 2.0.0-beta.2
